roles/wazuh/ansible-wazuh-agent/templates/var-ossec-etc-ossec-agent.conf.j2

```diff
@@ -30,7 +30,7 @@
     <config-profile>{{ wazuh_profile_ubuntu }}</config-profile>
     {% endif %}
     {% endif %}
-    {% if ansible_system == "Darwin" %}
+    {% if ansible_facts.system == "Darwin" %}
```

Comment:
wazuh-ansible master still uses ansible_system. Did you choose to make this change @Alex-Welsh?

Reply:
Could be worth proposing back upstream if so

Reply:
Yes, for me ansible_system is undefined because we unset `inject_facts_as_vars` for performance https://github.com/stackhpc/stackhpc-kayobe-config/blob/stackhpc/2024.1/etc/kayobe/ansible.cfg#L8